<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkDialog</class>
 <widget class="QDialog" name="MarkDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mark Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="colorLabel">
       <property name="text">
        <string>Background Color:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="colorLayout">
       <item>
        <widget class="QPushButton" name="colorButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Pick</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="colorDisplay">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>25</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::Box</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="startAddressLabel">
       <property name="text">
        <string>Start Address:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="startAddressEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="endAddressLabel">
       <property name="text">
        <string>End Address:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="endAddressEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="commentLabel">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QTextEdit" name="commentEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
